kernel/console/drivers/serial/serial_console.c

```diff
@@ -0,0 +1,25 @@
+// This file is part of CosmOS                                    *
+// Copyright (C) 2020 Tom Everett                                 *
+// Released under the stated terms in the file LICENSE            *
+// See the file "LICENSE" in the source distribution for details  *
+// ****************************************************************
+
+#include <console/drivers/drivers.h>
+#include <console/drivers/serial/serial_console.h>
+#include <dev/serial/serial.h>
+
+void serial_console_init(){	
+	return;
+}
+
+uint8_t serial_console_write(const char *c){
+    serial_write(c);
+}
+
+void serial_console_register(){
+	console_interfaces[CONSOLE_DRIVER_SERIAL].init = &serial_console_init;
+	console_interfaces[CONSOLE_DRIVER_SERIAL].setpos =0;
```

Comment:
I'll update console_setpos to add a check to ensure that the setpos function is defined for the driver before calling it, to avoid bad things happening if someone tries to run setpos on a console that doesn't have that implemented.  I should have done it from the start.

Reply:
Thanks!